ruleset/rules/0920-oracledb_rules.xml

```diff
@@ -1,31 +1,24 @@
 <!--
-  - OracleDB rules
-  - Created by Wazuh, Inc.
-  - Copyright (C) 2015-2021, Wazuh Inc.
-  - This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2.
+  Copyright (C) 2015-2021, Wazuh Inc.
 -->
 
-<group name="oracle,oracledb,">
+<!-- 
+  Rules for:
+    OracleDB ID: 89100 - 89199
+-->
```

Comment:
Software version should be specified just as in the decoders file

Reply:
Done